http/oidc/src/main/java/org/wildfly/security/http/oidc/RefreshableOidcSecurityContext.java

```diff
@@ -34,16 +34,18 @@ public class RefreshableOidcSecurityContext extends OidcSecurityContext {
     protected transient OidcClientConfiguration clientConfiguration;
     protected transient OidcTokenStore tokenStore;
     protected String refreshToken;
+    protected transient OidcHttpFacade facade;
 
     public RefreshableOidcSecurityContext() {
     }
 
-    public RefreshableOidcSecurityContext(OidcClientConfiguration clientConfiguration, OidcTokenStore tokenStore, String tokenString,
+    public RefreshableOidcSecurityContext(OidcClientConfiguration clientConfiguration, OidcHttpFacade facade, OidcTokenStore tokenStore, String tokenString,
```

Comment:
Just wondering, is it possible to pass in the cookie here instead of the facade?

Reply:
done